src/network/network.cpp

```diff
@@ -132,7 +134,8 @@ uint32 NetworkPacket::GetCommand()
 {
 	if (data->size() >= sizeof(uint32)) {
 		return ByteSwapBE(*(uint32*)(&(*data)[0]));
-	} else {
+	}
+	else {
```

Comment:
@Sixsmorphugus what are you merging exactly? You seem to have made a mess with the commits. You should rebase against develop not merge. And please refrain from reformatting irrelevant code - if you are going to do that, do that in a separate PR.

Instead of revert commits, can I suggest just amending them, rebasing them or checking out an older commit?

Also please use descriptive messages rather than "Intergrate fire with cave"